CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 4.0.1
+  - Added threadsafety; this plugin can now be used in pipelines with more than one worker.
```

Comment:
these all came together in a single PR, so I'm not sure how linking should work to align with our changelog standards

Reply:
What about something like: 
```

## 4.0.1
  - General improvements to code and docs [#29](https://github.com/logstash-plugins/logstash-filter-cipher/pull/29)
    - Added threadsafety; this plugin can now be used in pipelines with more than one worker.
    - Fixed a potential leak of the configured key into logs; the key is now only included if trace-level logging is enabled.
    - Fixed an issue where configurations that used invalid `mode` or `algorithm` settings could produce unhelpful error messages.
    - Fixed an issue where a bad payload could cause the plugin to crash; when exceptions are encountered, the offending event will now be tagged with `_cipherfiltererror`.
    - Improved documentation substantially

```

Reply:
Ah, ye olde nested bullets. I like it.